internal/pkg/db/memory/export-memory-database.go

```diff
@@ -13,33 +13,34 @@
  * or implied. See the License for the specific language governing permissions and limitations under
```

Comment:
Maybe the name should be export.go instead of export-memory-database.go, like it's done with core data.go and metadata.go